Miscellaneous bugs

flacks opened the pull request:
Co-authored-by: Gilbert Turner <grturner@5x5code.com>

flacks committed:
Fix updating car

Co-authored-by: Gilbert Turner <grturner@5x5code.com>

grturner committed:
Bug Fix: Logo
Logo link should work correctly and now routes you either to the home page or the landing page depending on whether or not a user is logged in.

grturner committed:
Membership form bugfix
Button was missing a click directive. Additionally the url the service was using is wrong. This will need to be revisited after model change pr is merged.

IkenoXamos commented on src/app/components/navbar/navbar.component.html:
```diff
@@ -1,8 +1,8 @@
 
-          <a class="navbar-brand logoRev" href="#">
-              <img class="imageLogo" src="../assets/revature-logo-600x219.png" routerLink="/" alt="The Revature logo.">
+          <a class="navbar-brand logoRev" routerLink={{link}}>
```
Should this instead be [routerLink]="link"?

flacks replied:
@grturner

flacks replied:
Since being logged-in reassigns navbar's `link` to the appropriate endpoint, property binding vs string interpolation accomplishes the exact same thing in this context, AFAICT.

IkenoXamos replied:
That is correct, but property binding is preferred in this context. Perhaps that is just my preference, but I usually find that string interpolation is used for innerHTML and property binding is used for attribute values.

IkenoXamos commented on src/app/components/profile-membership/profile-membership.component.html:
```diff
@@ -23,7 +23,7 @@ <h6>Status:</h6>
                     
             
-                <button type="submit" class="btn btn-primary myBtn">Save</button>
+                <button type="submit" (click)="updatesMembershipInfo()" class="btn btn-primary myBtn">Save</button>
```
Where does this method come from and what does it do?

flacks replied:
This event actually lets us call the function that updates the membership info, in the `profile-membership` controller.

IkenoXamos replied:
Alrighty, thank you.

grturner committed:
Requested change
Changed the routerlink on the revature logo in the navbar to use property binding instead of string interpolation
